app/managers.py

```diff
@@ -0,0 +1,42 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("app/cinema.sqlite")
```

Comment:
Consider closing the database connection when it is no longer needed to prevent resource leaks. You can implement a method to close the connection or use a context manager.